canonical-web-and-design/jaas-dashboard
# Add a confirmation dialog before submitting an action.

## Done

When trying to submit an action command a confirmation dialog will now pop up.

## QA

- Pull code
- Run `./run clean && ./run serve`
- Open http://0.0.0.0:8036/
- Select an action and submit, a confirmation dialog will show up
- If you hit cancel you should go back to the panel,
- If you hit submit it should submit the action (You can confirm this in the devtools network panel)

## Details

Fixes https://github.com/canonical-web-and-design/juju-squad/issues/1665

## Screenshots

![Screen Shot 2021-03-29 at 11 53 25 AM](https://user-images.githubusercontent.com/532033/112878627-66856380-9085-11eb-8ed2-dcd47d9a2a97.png)
